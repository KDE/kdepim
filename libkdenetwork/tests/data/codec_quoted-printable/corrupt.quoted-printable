Trailing spaces should be stripped:     
Here, only five trailing spaces should appear:    =20      
Trailing tabs should be stripped:			
Here, only five trailing spaces should appear:    =20			
Trailing mixture of tabs and spaces should be stripped: 	  	
Trailing mixture of tabs and spaces should be stripped:	 	  	
Tab, space, space, tab:	  =09	   	 	
Space, tab, tab, space: 		=20	  	  

Trailing spaces should=     
 be stripped:     
Trailing tabs should=			
 be stripped:			
Trailing mixture of tabs=	  	 	
 and spaces= 		 	   	
 should be stripped: 	  	
Tab, space, space, tab:	  	=	   	 	
Space, tab, tab, space: 		 =	  	  
The end.

A =3D wasn't properly encoded (should be kept): APE=MAN MAN=APE
A =3D wasn't properly encoded (lowercase): ape=man man=ape
Lowercase hexchars: =bb=a1=4b=44=45 =72=75=6c=65=7a=21=ab
Mixed-case hexchars: =Bb=A1=4B=44=45 =72=75=6C=65=7A=21=aB
A misplaced (unencoded =3D), followed by whitespace: =  not at end!
Two consecutive =3D at the end of the line: ==
Same, followed by trailing whitespace: ==   	 	 
A misplaced (unencoded =3D), as the ultimate character: =