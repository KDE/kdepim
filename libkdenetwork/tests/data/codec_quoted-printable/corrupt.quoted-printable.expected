Trailing spaces should be stripped:
Here, only five trailing spaces should appear:    
Trailing tabs should be stripped:
Here, only five trailing spaces should appear:    
Trailing mixture of tabs and spaces should be stripped:
Trailing mixture of tabs and spaces should be stripped:
Tab, space, space, tab:	  	
Space, tab, tab, space: 		 

Trailing spaces should be stripped:
Trailing tabs should be stripped:
Trailing mixture of tabs and spaces should be stripped:
Tab, space, space, tab:	  	Space, tab, tab, space: 		 The end.

A = wasn't properly encoded (should be kept): APE=MAN MAN=APE
A = wasn't properly encoded (lowercase): ape=man man=ape
Lowercase hexchars: »¡KDE rulez!«
Mixed-case hexchars: »¡KDE rulez!«
A misplaced (unencoded =), followed by whitespace: =  not at end!
Two consecutive = at the end of the line: =Same, followed by trailing whitespace: =A misplaced (unencoded =), as the ultimate character: =